content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -193,6 +193,84 @@ it does the following when a Pod is created:
+is enabled by default, which creates a ConfigMap that records the configmap
+creation date as the tracking date. With the feature enabled, each use of the
+legacy secret token updates the secret label `kubernetes.io/legacy-token-last-used`,
+with the latest usage date as the value.
+
+Starting from Kubernetes version 1.29, the legacy ServiceAccount token cleaner
+is enabled by default. It runs as part of `kube-controller-manager`
+and acts asynchronously. Every 24 hours, the cleaner checks if the current time
+has been a `speficied time` since the ConfigMap tracking date. If true, it loops
+the secret list and evaluates the secrets that have the type `kubernetes.io/service-account-token`.
+If the secret satisfies all the following conditions:
+- is auto-generated (secret bi-directionally referenced by the ServiceAccount)
+- is not mounted by pods
+- is not used in a `specified time` since the last-used/secret-creation date
+
+the cleaner marks the secret as invalid by adding a label `kubernetes.io/legacy-token-invalid-since`,
+with the current date as value. If the invalidated token is not used in a
+`specified time`, it will be cleaned by the cleaner.
+
+{{< note >}}
+All the `specificied time` above is default to be one year. Users can configure
+this value through `--legacy-service-account-token-clean-up-period` flag.
+{{< /note >}}
+
+If users use the invalidated auto-generated legacy token, the legacy token
+validator will
+- add an audit annotation for the key-value pair
+`authentication.k8s.io/legacy-token-invalidated`:`secretName/Namepace`,
+- increment the `invalid_legacy_auto_token_uses_total` metric count,
+- update the secret label `kubernetes.io/legacy-token-last-used` with the new date,
+- return an error indicating that the legacy token has been invalidated.
+
+If received this validation error, users can update the secret to remove the
+`kubernetes.io/legacy-token-invalid-since` label to temporarily allow use of
+this token.
+
+Here's an example of an auto-generated legacy token that has been marked with the
+`kubernetes.io/legacy-token-last-used` and `kubernetes.io/legacy-token-invalid-since`
+labels:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: build-robot-secret
+  namespace: default
+  labels:
+    kubernetes.io/legacy-token-last-used: 2022-10-24
+    kubernetes.io/legacy-token-invalid-since: 2023-10-25
+  annotations:
+    kubernetes.io/service-account.name: build-robot
+type: kubernetes.io/service-account-token
+```
+
+{{< note >}}
+The difference between the manually created legacy token and
+the auto-generated legacy token is that only auto-generated legacy token is
+bi-directionally referenced by the ServiceAccount, like:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: build-robot
+  namespace: default
+secrets:
+  - name: build-robot-secret
+```
+
+If you want to use the manually created legacy token, please do not add the
+secret name to the `Secrets` field of the service account.
```

Comment:
Again, please check your whole PR for spelling ...

```suggestion
Secret name to the `secrets` field of the ServiceAccount.
```

Reply:
Checked and revised.